iOSClient/Media/NCMediaTabbarSelect.swift

```diff
@@ -25,8 +25,7 @@ import UIKit
 import SwiftUI
 
 protocol NCTabBarSelectDelegate: AnyObject {
```

Comment:
Please rename it to NCMediaTabBarSelect + the delegate. There are multiple tab bars now